op-node/rollup/driver/state.go

```diff
@@ -290,7 +290,7 @@ func (s *Driver) eventLoop() {
 			s.log.Debug("Derivation process step", "onto_origin", s.derivation.Origin(), "attempts", stepAttempts)
 			err := s.derivation.Step(context.Background())
 			stepAttempts += 1 // count as attempt by default. We reset to 0 if we are making healthy progress.
-			if err == io.EOF {
+			if err == io.EOF || (err != nil && errors.Is(err, derive.EngineSyncing)) {
```

Comment:
Is this equivalent to 

```suggestion
			if err == io.EOF || errors.Is(err, derive.EngineSyncing) {
```

??